docs/taskruns.md

```diff
@@ -398,6 +398,8 @@ If a `TaskRun` runs longer than its timeout value, the pod associated with the `
 means that the logs of the `TaskRun` are not preserved. The deletion of the `TaskRun` pod is necessary in order to
 stop `TaskRun` step containers from running.
 
+The above `timeout` is inclusive of the time used to execute the `TaskRun`'s `retries`.
```

Comment:
similar question to above: still not sure if this is the total time for taskrun + retries, or each retry will reset the counter

Reply:
I disagree that this should be the case. A `Task` author has no knowledge about retries, and when they set a timeout is definitely specific to the single execution of the `Task`.
A `TaskRun` may override that value to a different one specific to the runtime environment (slower/faster execution nodes for instance). 

When the `Task` is executed within a `PipelineRun` with retries, the `PipelineRun` author may want a mechanism to control the timeout for a `TaskRun` and its retries but I don't think we provide that today and I would see that as a new feature.

We do provide today a way for the `PipelineRun` author to control the maximum amount of time that can be spent in the "non-finally" part of the `Pipeline` (i.e. `timeouts.tasks`) which can be used to ensure that `finally` will run.